pkg/webhook/resources/virtualmachine/mutator.go

```diff
@@ -231,9 +267,24 @@ func generateMemoryPatch(vm *kubevirtv1.VirtualMachine, mem *resource.Quantity,
 		if err != nil {
 			return patchOps, err
 		}
+		// already checked on validator: reservedMemory < mem
+		if reservedMemory.Value() >= mem.Value() {
+			return patchOps, fmt.Errorf("reservedMemory cannot be equal or greater than limits.memory %v %v", reservedMemory.Value(), mem.Value())
+		}
+	} else {
+		// if user does not specify the value, then compute it dynamically
+		computeReservedMemory(mem, &reservedMemory)
 	}
 	guestMemory := *mem
 	guestMemory.Sub(reservedMemory)
+	if guestMemory.Value() < 268435456 {
```

Comment:
For better readability
```suggestion
	if guestMemory.Value() < 256*1024*1024 {
```
or
```suggestion
	if -1 == guestMemory.Cmp(resource.ParseQuantity("256Mi")) {
```